### ISSUE TITLE
says roblox isnt installed when it is

### ISSUE THREAD

**Issue by 3raden** [pogpoggers/roblox-hwid-ban-remover#2]
help me lol

**Comment by pogpoggers** [comment]
Is ROBLOX Player installed? this program will unban the client, not studio.
if installed, try running this program as admin.

**Comment by 3raden** [comment]
yeah roblox player is installed. i tried using it with administrator. still the same
![image](https://user-images.githubusercontent.com/64098772/135563101-e0cde49a-a35e-4ef5-8346-69445073f6c7.png)
